leap_c/mpc.py

```diff
@@ -425,7 +438,12 @@ def _solve_shared(
             solve_stats["sqp_iter"] += solver.get_stats("sqp_iter")
             solve_stats["qp_iter"] += solver.get_stats("qp_iter").sum()  # type:ignore
             solve_stats["time_tot"] += solver.get_stats("time_tot")
-        solve_stats["first_solve_status"] = [solver.status]
```

Comment:
I would also log the first solve status, not remove it (note that this is bugged here, this should be the status before the reattempt)

Reply:
But I strongly agree that the status logging should happen here in the MPC, I don't know why I didn't think of it myself so far 👍 .